perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/SearcherMethod.java

```diff
@@ -90,5 +91,29 @@
 			return ac.getSearcher().getFacilities(ac.getSession(),
 					parms.read("attributesWithSearchingValues", LinkedHashMap.class));
 		}
+	},
+
+	/*#
+	 * This method get Map of Attributes with searching values and try to find all resources, which have specific attributes in format.
+	 * Better information about format below. When there are more than 1 attribute in Map, it means all must be true "looking for all of them" (AND)
+	 *
+	 * @param sess perun session
```

Comment:
Session parameter is implicit -> not send to API from outside. Please remove it from RPC API javadoc and do the same for previously implemented getFacilities() method. Thank you.